chartjs-plugin-annotation.js

```diff
@@ -849,25 +853,25 @@ module.exports = function(Chart) {
 				ya = padHeight + view.labelYAdjust;
 				xa = (width / 2) + view.labelXAdjust;
 				ret.y = view.y1 + ya;
-				ret.x = (isFinite(line.m) ? line.getY(ret.y) : view.x1) - xa;
+				ret.x = (isFinite(line.m) ? line.getX(ret.y) : view.x1) - xa;
 			break;
-			
+
 			// bottom align
 			case view.mode == verticalKeyword && view.labelPosition == "bottom":
 				ya = height + padHeight + view.labelYAdjust;
 				xa = (width / 2) + view.labelXAdjust;
 				ret.y = view.y2 - ya;
 				ret.x = (isFinite(line.m) ? line.getX(ret.y) : view.x1) - xa;
 			break;
-			
+
 			// left align
 			case view.mode == horizontalKeyword && view.labelPosition == "left":
 				xa = padWidth + view.labelXAdjust;
 				ya = -(height / 2) + view.labelYAdjust;
 				ret.x = view.x1 + xa;
 				ret.y = line.getY(ret.x) + ya;
 			break;
-			
+
```

Comment:
Please revert all changes to chartjs-plugin-annotation.js and chartjs-plugin-annotation.min.js. We will compile these when we make a release.